Automattic/wp-calypso
# Clean up Limit Global Styles logic

#### Proposed Changes

Cleans up some technical debt from the logic that limits Global Styles:
- The `theme_json_user` filter is no longer used to block styles since WP.com no longer loads the Gutenberg version that triggered it
- Remove usages of `_Gutenberg` classes since they are not intended to be used externally.

#### Testing Instructions

- Apply these changes to your sandbox: `install-plugin.sh editing-toolkit update/clean-up-limit-global-styles`
- Go to https://wordpress.com/start and create a new free site
- Add the `wpcom-limit-global-styles` blog sticker to the new site from the Blog RC
- Sandbox the new site
- Go to Appearance > Editor
- Open the Global Styles sidebar
- Change some styles
- Click on the "Save" button
- Visit the frontend
- Make sure only the styles from the default theme variation are visible
- Make sure the launchbar contains a "Styles hidden" action
